editor/views/overview.py

```diff
@@ -1,2 +1,15 @@
+import streamlit as st
+
+from core.state import CurrentStep
+from core.state import Metadata
+from utils import set_form_step
+
+
 def render_overview():
-    pass
+    meta = st.session_state[Metadata]
+    st.subheader("Name:")
+    st.markdown(meta.name)
```

Comment:
Should you put the `st.text_input` here and remove them from the `Metadata` page?